### PR TITLE
Bug fixes

### LINKED ISSUE #85: Improve treatment for unchanged costs
**What needs to be done? Why does it need to be done?**
As seen in the screenshot below, when the outcomes show that there's no change in population or costs, there is inconsistent treatment between the two. We see "No change in population," a nice human readable string, but we just see a kind of weird "$0K reduced costs" with a down arrow that's not accurate.

We should change the costs treatment to be the same as the population treatment: no arrow present, no k (it should just show "$0"), and the text should read "No change in cost"

**Additional context**
Screen Shot 2021-07-22 at 14 14 02

### LINKED ISSUE #87: Chart tooltips don't show up in mobile
**Describe the bug**
The chart tooltips, which appear on hover for desktop devices, should appear on click for mobile devices. At the moment, this doesn't appear to be working. Clicking on the chart does not show a tooltip. It only shows the dotted line between data points treatment.

**To Reproduce**
1. Open the app in Chrome
2. Open up developer tools and change the layout to a mobile device
3. Scroll down to the chart
4. Click on any data points in the chart
5. See that the tooltip does not show up

**Expected behavior**
The tooltip should appear when you click on data points in the chart.

**Screenshots**
Screenshots revealing sensitive information _must not_ be shared. If a screenshot is helpful in diagnosing and resolving the bug, please send this directly to a contact on the Recidiviz team.

**Desktop (please complete the following information):**
 - OS [e.g. Windows 10]: None
 - Browser [e.g. chrome, edge]: None
 - Version [e.g. 22]: None

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone6]: All?
 - OS: [e.g. iOS8.1]: All?
 - Browser [e.g. stock browser, safari]: All?
 - Version [e.g. 22]: All?

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by nasaownsky**
## Description of the change

1. Enabled tooltips on mobile
2. Improved treatment for unchanged costs

## Type of change

- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #87 , closes #85 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

**Review by jessex — CHANGES_REQUESTED** [review]
There's a weird inconsistency here, @nasaownsky, and I think it probably has to do with rounding. In the two screenshots below, you can see that the fix here _didn't work_ when projection length was set to 4 years, but _did work_ when it was set to 5 years.

Screen Shot 2021-07-26 at 10 56 44 AM
Screen Shot 2021-07-26 at 10 56 55 AM

**Comment by jessex** [comment]
@cawarren I had filed an issue late last week to make tooltips on the chart work in mobile, but seeing Ilya's PR this morning reminds me that in V1 of the app, we explicitly disabled tooltips on mobile because of their unpredictable UX. Do you want to keep tooltips disabled in mobile for this version, too? It will be a simple thing to revert Ilya's one-line enablement here.

**Comment by cawarren** [comment]
Great catch, I'd forgotten about that. Let's revert, just to minimize complexity / risk right now. If we get a lot of requests for them we look further into the mobile behavior / potential workarounds.

File: src/SupervisionSuccess/model/produceProjections.js [hunk 1]
```diff
@@ -171,7 +175,7 @@ function produceProjections(
 
   return {
     chartData,
-    savings: totalSavings > 0.0001 || totalSavings < 0 ? Number(totalSavings.toFixed(6)) : 0,
+    savings: Number(totalSavings.toFixed(3)),
```
Comment (nasaownsky) [inline src/SupervisionSuccess/model/produceProjections.js:178]: @jessex @cawarren The problem with 0 behavior comes from total savings and its accuracy, but error does not exceed 0.0005 (for now, at least). At the same time we do not use numbers smaller than 0.001 for savings, so using `toFixed(3)` instead `toFixed(6)` has the meaning. We can do so, or use the condition commented above.

Comment (jessex) [reply src/SupervisionSuccess/model/produceProjections.js:174]: I think this approach you have now seems good for the use case. Thanks, @nasaownsky!

**Review by jessex — APPROVED** [review]
Good to go!

**Pull request merged**